packages/front-end/services/DefinitionsContext.tsx

```diff
@@ -75,6 +76,7 @@ const defaultValue: DefinitionContextValue = {
   projects: [],
   factTables: [],
   factMetrics: [],
+  _factMetricsIncludingArchived: [],
```

Comment:
We should do a similar thing for fact tables.  There are a few places in the UI where we ask you to select a fact table.  If it's archived, it should not appear in the select box.  Easiest is to do what we do with metrics where `definitions.factTables` excludes them by default.